Gemfile

```diff
@@ -31,6 +31,9 @@ gem "rake", "10.1.1", :require => false
 # Hiera has an unbound dependency on json_pure
 # json_pure 2.0.2+ officially requires Ruby >= 2.0, but should have specified that in 2.0
 gem 'json_pure', '~> 1.8', :require => false
+# i18n support (gettext-setup and dependencies)
+gem 'gettext-setup', '>= 0.8', '< 1.0', :require => false
```

Comment:
I don't think so.. The only issue I can see if we do some sort of version equality detection across the gem paths of the two projects but I don't think we do anything like that- it's just assumed we have compatible versions in both where needed. There isn't anything that changed between 0.8 and 0.10 that would be used in the context of Puppet server having loaded puppet's code, so dependency mismatch aside it should load/operate fine.